### DISCUSSION TITLE
NoScript with Ad Nauseum?

### DISCUSSION THREAD

**Discussion by 381outrageousDoors** [dhowe/AdNauseam#2424] [Q&A]
Hi, is it recommended to turn off NoScript if using this or use NoScript with recommended settings? Assuming disable but can't find anything on it, so thought I'd ask

**Comment by mneunomne** [comment]
This is an option we inherit from uBlock Origin, it should be only used in fringe cases for your security, since nowdays most websites need JavaScript to run.
